packages/alfa-dom/test/node/attribute.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { None } from "@siteimprove/alfa-option";
+import { Parser } from "@siteimprove/alfa-parser";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { test } from "@siteimprove/alfa-test";
+
+import { Attribute } from "../../src";
+
+const makeAttribute = (str: string) => Attribute.of(None, None, "dummy", str);
+
+test("parse space separated token list", (t) => {
```

Comment:
```suggestion
test("#tokens() parses a space separated token list", (t) => {
```

We should use proper sentences for test case descriptions and make sure to note which method/function is under test.